addons/website/views/website_templates.xml

```diff
@@ -535,6 +535,9 @@
 </template>
 
 <template id="template_header_sidebar" inherit_id="website.layout" name="Template Header Sidebar" active="False">
+    <xpath expr="//header[contains(@t-attf-class, 'o_header_fixed')]" position="attributes">
+         <attribute name="t-attf-class" remove="o_header_fixed" separator=" "/>
+    </xpath>
```

Comment:
-> Select "Scroll" scolling effect
-> Select the Sidebar template
-> Crash

The xpath you do here obviously fails since it only works when the header is fixed...

Reply:
Hello @qsm-odoo 
Thanks for review.
I have made new changes through javascript.
when selector '.o_header_fixed' adds 'o_header_affixed' and transform properties.
scrolling becomes problematic when we have side bar menu due to this adition.
So, I prevent to append when header is side bar. now, Its working.
Thanks in advance for your inputs!